### DISCUSSION TITLE
Next quark design 邀您共建 🚀

### DISCUSSION THREAD

**Discussion by xxxsf** [hellof2e/quark-design#228] [Announcements]
## Task List
任务列表

日常任务：

- [x] vue 2.6 + tsx + comosition api 下 slot 属性问题  @sanqi-med 

长线任务：

- [x] quark 引擎 @xsf0105 
- [x] quark playground @dyf19118 @sanqi-med 
- [ ] quark playground & npm 一站式发布平台
- [x] quark 引擎官网 @xsf0105 
- [ ] YOU-DONT-NEED-React-Or-Vue @nieyt
- [x] Create-quark-app for components @xsf0105 
- [ ] Create-quark-app for app
- [x] quark md2page（markdown 生成 路由页面）@nieyt @xsf0105 
- [x] quark-design 文档设计探索，如何支持各种技术栈的文档展示
- [x] quark-design/packages/quark 文件名改为 quark-design/packages/quarkd

相关工程分布在 hellof2e 组织下，有问题可留言咨询，哈啰内部开发者可直接钉钉咨询。

**Comment by yoyo837** [comment]
目前是1.x要直接跳到4.x吗? 另外没有看到 v4分支呢?

**Reply by xxxsf** [reply]
描述有误，已修改

**Comment by yoyo837** [comment]
小弟目前在项目前期阶段, 过段时间会积极参与贡献, 感谢🙏!

**Reply by xxxsf** [reply]
👏欢迎参与，希望能和大家一起携手同行。
ps：quark 不止组件库，还有其它相关生态，后面会慢慢和大家见面～

**Comment by chenyulun** [comment]
mark,之前搞过一段时间stencil，也看过源码，到时候看能不能借鉴弄点优秀的特性过来（异步加载组件什么的）

**Reply by xxxsf** [reply]
stencil 200多issue、热更新非常慢，还存在黑盒，推荐使用 Quarkc。原生，轻量

**Reply by chenyulun** [reply]
请教一下，他热更新慢的根本原因是什么，源码不都是开源的吗，怎么存在黑盒？

**Reply by xxxsf** [reply]
你去玩一玩 Stencil 就知道啦，当然你觉得能接受就没问题
他有自己的编译工具，非主流
